docs/source/api/v2/servers_id_deliveryservices.rst

```diff
@@ -239,3 +239,81 @@ Response Structure
 
 
 .. [#tenancy] Only the :term:`Delivery Services` visible to the requesting user's :term:`Tenant` will appear, regardless of their :term:`Role` or the :term:`Delivery Services`' actual 'server assignment' status.
+
+``POST``
+========
+Assign an arbitrary number of delivery services to a single server.
+
+:Auth. Required: Yes
+:Roles Required: "admin" or "operations"
+:Response Type:  Object
+
+Request Structure
+-----------------
+.. table:: Request Path Parameters
+
+	+------+----------+---------------------------------------------------------------------------------------------+
+	| Name | Required | Description                                                                                 |
+	+------+----------+---------------------------------------------------------------------------------------------+
+	| id   | Yes      | The integral, unique identifier of the server that you want to assign delivery services to. |
```

Comment:
path parameter is listed as "ID" not "id"

Reply:
Capitalized path parameter in 8fe4ab7d06.